package.json

```diff
@@ -696,22 +696,16 @@
                     "type": "boolean",
                     "description": "Optional flag to determine whether diagnostic engine logs should be logged to the output window.",
                     "default": false
-                  },
-                  "trace": {
-                    "type": "boolean",
-                    "description": "Optional flag to determine whether diagnostic adapter command tracing should be logged to the output window.",
-                    "default": false
-                  },
-                  "traceResponse": {
-                    "type": "boolean",
-                    "description": "Optional flag to determine whether diagnostic adapter command and response tracing should be logged to the output window.",
-                    "default": false
                   }
                 }
               },
               "pipeTransport": {
                 "description": "When present, this tells the debugger to connect to a remote computer using another executable as a pipe that will relay standard input/output between VS Code and the .NET Core debugger backend executable (clrdbg).",
                 "type": "object",
+                "required": [
+                  "pipeProgram",
```

Comment:
>"pipeProgram", [](start = 18, length = 14)

We might want to leave `pipeProgram` optional since it is possible to specify it for each OS rather than using a common one on all OSs